src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -23,14 +24,12 @@ public class CrossTenantQueryService {
   private final SimpleHttpClient ecsClient;
   private final FolioExecutionContext executionContext;
 
-  public List<String> getTenantsToQuery(UUID entityTypeId) {
```

Comment:
If you watch the logs when doing just about anything in FQM, you'll notice that they are just FLOODED with log messages about loading the same entity types from the DB over and over again. I think a huge part of that is that we tend to pass IDs around, so I went ahead and made this change to cut down on that at least a little. Caching entity type definitions will make a much bigger difference, but this change still seemed worthwhile